src/navigation/MainTabNavigator.tsx

```diff
@@ -90,17 +90,20 @@ export const MainTabNavigator = () => {
 
   if (!tabConfigs || loading) return <LoadingSpinner loading />;
 
+  const backgroundColor =
```

Comment:
what about `const { inactiveBackgroundColor: backgroundColor } = tabRoutes || tabNavigatorConfig;`? does this work?